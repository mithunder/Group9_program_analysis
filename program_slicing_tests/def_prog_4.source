module def_prog_4 :
a := 42;
do (z > 0) -> a := 30; y := 0; d := a+1; z := d * 3
[] (b - 0 + 3 > 0) -> read c
od;
@programslicing="true"
x := a
end

/*Note: Version where S2 does not reach P.*/
