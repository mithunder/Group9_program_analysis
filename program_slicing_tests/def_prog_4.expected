module def_prog_4 :
a := 42;
do (z > 0) -> a := 30; d := a+1; z := d * 3
od;
x := a
end
